metrics/web/lib/auth/presentation/models/user_profile_model.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+import 'package:metrics/auth/domain/entities/theme_type.dart';
+
+/// A class that represents a user profile model used to
+/// transfer data between [ChangeNotifier]s.
+class UserProfileModel extends Equatable {
+  /// A unique identifier of the user profile.
```

Comment:
Please, prefer using `this` instead of `the` as an [effective dart](https://dart.dev/guides/language/effective-dart/documentation#prefer-using-this-instead-of-the-to-refer-to-a-members-instance) suggests.